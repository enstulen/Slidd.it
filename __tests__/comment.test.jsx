import renderer from 'react-test-renderer';
import React from 'react';
import { shallow } from 'enzyme';
import Comment from '../imports/ui/components/comment.jsx';

describe('Comment component', () => {
  it('Renders correctly', () => {
    const rendered = renderer.create(
      <Comment key={123} comment="hei" createdAt={new Date()} />
    );
    expect(rendered.toJSON()).toMatchSnapshot();
  });
  it('Correct length', () => {
    const wrapper = shallow(<Comment key={123} comment="hi" createdAt={new Date()} />);

    expect(wrapper.find('.right').length).toEqual(1);
    expect(wrapper.find('.left').length).toEqual(1);
  });
});
