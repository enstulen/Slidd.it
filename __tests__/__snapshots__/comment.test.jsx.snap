// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comment component Renders correctly 1`] = `
<tr>
  <td
    className="right"
  >
     
  </td>
  <td
    className="left"
  >
     
    11
    :
    15
        
    4/22/2017
     
  </td>
</tr>
`;
